### DISCUSSION TITLE
How to hide the arrow in th Dropdown component?

### DISCUSSION THREAD

**Discussion by goldwind-ting** [ant-design/ant-design#38749] [Q&A]
```js
<Dropdown menu={{ items }} trigger={['click']}>
      <a onClick={(e) => e.preventDefault()}>
          <Space>
              File
              <DownOutlined />
          </Space>
      </a>
 </Dropdown>
```
![image](https://user-images.githubusercontent.com/63939636/202859916-f120981b-da8f-41e4-8cea-71542dd5e4f0.png)

**Comment by natashaamin** [comment]
`arrow={false}`



arrow | Whether the dropdown arrow should be visible
-- | --

**Reply by goldwind-ting** [reply]
It does not work.

**Reply by goldwind-ting** [reply]
please refer to the [source](https://gist.github.com/goldwind-ting/75b9a9a1f81a2bc5425d79578d78f5e6#file-gistfile1-txt-L30).

**Reply by natashaamin** [reply]
Isnt <DownOutlined> the arrow icon? Just remove that.

**Reply by goldwind-ting** [reply]
I don't use arrow icon, just like this:
```
<Dropdown menu={{ items }} trigger={['click']} arrow={false}>
        <a onClick={(e) => e.preventDefault()}>
            <Space>
                File
                <DownOutlined />
            </Space>
        </a>
    </Dropdown>
```
do you mean remove what?

**Reply by natashaamin** [reply]
Remove Downoutlined. Thats the icon
